packages/adblocker-electron/rollup.config.ts

```diff
@@ -34,4 +34,22 @@ export default [
       }),
     ],
   },
+  {
+    external: ['electron'],
+    input: './build/es6/content.js',
+    output: [
+      {
+        file: './dist/content.min.js',
+        format: 'cjs',
+        sourcemap: true,
+      },
+    ],
+    plugins: [
+      resolve(),
+      compiler({
+        language_out: 'NO_TRANSPILE',
+        warning_level: 'DEFAULT',
+      }),
+    ],
+  },
```

Comment:
Wondering what would be most convenient from a usage perspective. Do you think that having the two bundles `content.js` and `main.js` at the root (instead of `dist/` folder) would be desirable to allow importing like:

```js
import content from '@cliqz/adblocker-electron/content';
import main from '@cliqz/adblocker-electron';
```

Reply:
The `content.js` file is used for injecting preload into Electron's session.

Reply:
```typescript
ses.setPreloads(['./content.js']);
```

Reply:
I forgot about that, thanks for the reminder. Shouldn't the `output` of bundling be called `content.js` instead of `content.min.js` though?